src/Maestro/Maestro.Web/Api/v2019_01_16/Models/BuildTime.cs

```diff
@@ -6,15 +6,17 @@ namespace Maestro.Web.Api.v2019_01_16.Models
 {
     public class BuildTime
     {
-        public BuildTime(int defaultChannelId, double officialTime, double prTime)
+        public BuildTime(int defaultChannelId, double officialTime, double prTime, int goalTime)
         {
             DefaultChannelId = defaultChannelId;
             OfficialBuildTime = officialTime;
             PRBuildTime = prTime;
+            GoalTime = goalTime;
         }
 
         public int DefaultChannelId;
         public double OfficialBuildTime;
         public double PRBuildTime;
+        public int GoalTime;
```

Comment:
What unit is this time in?  It should probably have the unit specified in the variable name.

Reply:
It's minutes, the same as the GoalTime api. I can update it.